CONTRIBUTING.md

```diff
@@ -27,18 +27,82 @@ Cum sociis [natoque](https://www.google.com/) penatibus et magnis dis.
 ```
 
 
+### Short code notice
+
+The notice shortcode shows 4 types of disclaimers.
+
+
+#### Note
+
+```
+{{% notice note %}}
+A notice disclaimer
+{{% /notice %}}
+```
+
+#### Info
+
+```
+{{% notice info %}}
+An information disclaimer
+{{% /notice %}}
+```
+
+#### Tip
+
+```
+{{% notice tip %}}
+A tip disclaimer
+{{% /notice %}}
+```
+
+#### Warning
+
+```
+{{% notice warning %}}
+A warning disclaimer
+{{% /notice %}}
+```
+
+
+### Short code tabs
+
+Choose which content to see across the page. Very handy for providing code snippets for multiple languages or 
```

Comment:
> Choose which content to see across the page.

Not quite sure what you mean by that? The short code tab allows you to group content into tabs.

Reply:
Just using the original description from https://learn.netlify.app/en/shortcodes/tabs/
Will fix that ...

Reply:
Done